dfir_lang/src/graph/meta_graph.rs

```diff
@@ -1013,6 +1013,46 @@ impl DfirGraph {
                                     singletons_resolved.clone(),
                                 );
 
+                            let source_tag = 'a: {
+                                if let Some(tag) = self.operator_tag.get(node_id).cloned() {
+                                    break 'a tag;
+                                }
+
+                                #[cfg(nightly)]
+                                if proc_macro::is_available() {
+                                    let op_span = op_span.unwrap();
+                                    break 'a format!(
+                                        "loc_{}_{}_{}_{}_{}",
+                                        op_span
+                                            .source_file()
+                                            .path()
+                                            .display()
+                                            .to_string()
+                                            .replace(|x: char| !x.is_alphanumeric(), "_"),
+                                        op_span.start().line(),
+                                        op_span.start().column(),
+                                        op_span.end().line(),
+                                        op_span.end().column(),
+                                    );
+                                }
+
+                                format!(
+                                    "loc_nopath_{}_{}_{}_{}",
+                                    op_span.start().line,
+                                    op_span.start().column,
+                                    op_span.end().line,
+                                    op_span.end().column
+                                )
+                            };
+
+                            let fn_ident = format_ident!(
+                                "{}__{}__{}",
+                                ident,
+                                op_name,
+                                source_tag,
+                                span = op_span
+                            );
```

Comment:
Do you think re-using this in three different fn calls will cause any confusion?

Reply:
Since the only purpose of wrapping in a function is to show up in perf, I _think_ it should be okay; the type signatures are also different so we won't confuse them.